pkg/authenticator/authenticator.go

```diff
@@ -112,6 +112,25 @@ func (auth *Authenticator) Login() error {
 
 	log.InfoLogger.Printf(log.CAKC007I, auth.Config.Username)
 
+	// The cert file may be present from a previous login request
+	if !auth.CertFileExist() {
```

Comment:
what if the cert file exists, but is no longer valid? don't we have a method to check if the cert is valid in one of these packages? should that be invoked here?